package.json

```diff
@@ -106,18 +105,30 @@
     "eslint": "eslint src",
     "test": "TZ=Asia/Taipei node scripts/test.js --env=jsdom",
     "coverage": "yarn run test -- --coverage",
-    "coveralls": "NODE_ENV=development cat coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js"
+    "coveralls": "NODE_ENV=development cat coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js",
+    "prettier:all": "prettier --write 'src/example.js' 'src/components/**/*.js' 'config/**/*.js'"
   },
   "lint-staged": {
-    "*.js": [
-      "jest --findRelatedTests",
-      "eslint --max-warnings 0"
-    ]
+    "linters": {
+      "*.js": [
+        "yarn prettier --write",
```

Comment:
will these work with npm too?

Reply:
Good point.
I use `$npm_execpath` instead to be compatible with yarn/npm.

Reply:
Sorry, I found the `$npm_execpath` way is not working in lint-staged. So I'll revert it to yarn.
And it has to run with `yarn` in current time.